### PR TITLE
DOC: Added announcement for PCA

### PR DESCRIPTION
**Pull request by bashtage**
Added announcement for class-based PCA implementation with a listing
of new features.

### CODE REVIEW

**Comment by bashtage** [comment]
Closes #1238

File: docs/source/release/version0.7.rst [hunk 1]
```diff
@@ -11,10 +11,39 @@ Release summary
...
+
+A new class-based Principal Component Analysis has been added.  This
+class replaces the function-based PCA that previously existed in the
+sandbox.  This change bring a number of new features, including:
+
+* Options to control the standardization (demeaning/studentizing)
+* Scree plotting
+* Information criteria for selecting the number of factors
+* R-squared plots to assess component fit
+* NIPLS implementation when only a small number of components are required and the dataset is large
```
Comment (josef-pkt) [inline docs/source/release/version0.7.rst]: missing A in NIPALS ?

File: docs/source/release/version0.7.rst [hunk 2]
```diff
@@ -11,10 +11,39 @@ Release summary
...
+   dta = data[columns]
+   dta = dta.dropna()
+
+   pca_model = PCA(dta.T, standardize=False, demean=True)
+
+   pca_model.plot_scree()
 
-Summary of changes with example.
+*Note* : For backward compatability, the sandbox function has not been
+removed. However, it is just a think wrapper around the class-based PCA
```
Comment (josef-pkt) [inline docs/source/release/version0.7.rst]: think -> thin
I don't remember that it was replaced by a wrapper function.

**Comment by josef-pkt** [comment]
Thanks,

I will be back to merging in a few hours.

**Comment by bashtage** [comment]
You are right - it is not a wrapper.  I fixed this comment to reflect that there is a new function-based version that has the same signature as the sandbox version.

**Comment by josef-pkt** [comment]
Thanks @bashtage 
merging

**Comment by josef-pkt** [comment]
BTW: adding `[skip ci]` to the commit message avoids the travis test run, if a commit or push doesn't affect code or only code that is not run by unit tests

**Comment by bashtage** [comment]
When do the dev docs get built?  I would like to see it and check that the layout is right.

I added a [skip ci] in one of the amended commits :)

**Pull request merged**